microsoft/PowerToys
# [PT Run] Run dialog now has monitor positioning options

## Summary of the Pull Request

**What is this about:**
Allow users to specify where to show the Run launcher window
**What is include in the PR:** 
Options to show window on: follow cursor, primary monitor, or a specified monitor (via index)
**How does someone test / validate:** 
- Run PowerToys
- change the `Position` setting in the Run settings window to one of the options
- press hotkey to trigger the window and check it appears in the correct place
## Quality Checklist

- [x] **Linked issue:** #1607

## Contributor License Agreement (CLA)
A CLA must be signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/PowerToys) and sign the CLA.


ToDo:
- [x] validate index input - limit to number of monitors detected
- [x] fix radio buttons not showing state on initial settings load


Screenshots:

![image](https://user-images.githubusercontent.com/2855069/108067909-58d9c880-7059-11eb-8118-151fa9ef4837.png)
![image](https://user-images.githubusercontent.com/2855069/108067992-7313a680-7059-11eb-815f-806cfe0db727.png)
![image](https://user-images.githubusercontent.com/2855069/108068019-7ad34b00-7059-11eb-9880-8d546393c51a.png)
